### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by iSentrie** [renzuzu/renzu_nitro#11]
I have tested resource and this is what I found:

1. On purge there are screen effects which should not show up.
2. Smoke or whatever it is gets cut off instead of vaporing in air
3. If you hit `Shift+W` and the same time that smoke will show instead of nitro flames.

https://user-images.githubusercontent.com/29688478/196724426-2f1d50e8-f1e4-4cad-9f78-7d4df6747e44.mp4

**Comment by renzuzu** [comment]
i just tweak the client and probably solve the 1-4.
the #5 issue is i cant replicated by just spamming SHIFT. probably i fixed it on 1-4

**Comment by iSentrie** [comment]
Yes, indeed much better and no errors. :)
Next what I would do is adjusting purge itself, from tires it doesn't look very well and the amount of it is a bit too much, isn't?

**Comment by iSentrie** [comment]
did some tests, you can check it out
```lua
purge1 = StartParticleFxLoopedOnEntityBone_2('ent_sht_steam', vehicle, 0.25, 0.125, 0.15, 65.0, 30.0, 0.0, carEngine, 0.75, 0, 0, 0)
purge2 = StartParticleFxLoopedOnEntityBone_2('ent_sht_steam', vehicle, -0.25, 0.125, 0.15, 65.0, -30.0, 0.0, carEngine, 0.75, 0, 0, 0)
```

**Comment by renzuzu** [comment]
yes its my new plan actually to look for better implementation of smokes or purges position.

im thinking to do a UI to edit the position. and yes with smoke color
